classes/log/EmailLogDAO.php

```diff
@@ -157,7 +159,7 @@ public function insertObject($entry)
                 $entry->getRecipients(),
                 $entry->getCcs(),
                 $entry->getBccs(),
-                $entry->getSubject(),
+                $this->_truncateEmailSubjectLine($entry->getSubject()),
```

Comment:
I'd suggest using [`Str::limit`](https://laravel.com/docs/11.x/strings#method-str-limit) rather than writing a custom function. (This will also solve the problem below that you're using the constant in some parts, but still have a hard-coded `252` in others.)

Reply:
updated